### ISSUE TITLE
PyCharm plugin text is large, does not resize

### ISSUE THREAD

**Issue by cf-home** [pieces-app/support#240]
### Software

JetBrains

### Operating System / Platform

macOS

### Your Pieces OS Version

9.0.5

### Early Access Program

- [ ] Yes, this is related to an Early Access Program feature.

### Kindly describe the bug and include as much detail as possible on what you were doing so we can reproduce the bug.

In the attached, the text size is large and gets cut off.  If I make the window smaller, even more text gets cutoff.  There is no left/right scrollbar.  I would prefer if the window could be resized.  My PyCharm version is 2024.1.2 Community Edition.
![Screenshot 2024-06-03 at 5 47 17 AM](https://github.com/pieces-app/support/assets/160560521/328ab870-c806-4e3a-beae-dafefb1ce501)

**Comment by caleb-at-pieces** [comment]
Hello @cf-home ! Thank you for bringing this to our attention.

This is generally not expected behavior. There is not supposed to be any horizontal overflow in the Copilot window.

I have a few questions to gather some information about this:

First and foremost, does restarting PyCharm resolve this issue?

Does this occur while plugging in / unplugging displays? Or maybe resizing PyCharm itself?

Do you see any logs in the Copilot Webview console? Here are some instructions to open this:
- command + shift + a
- select 'registry...'
- search 'ide.browser.jcef.contextMenu.devTools.enabled' & check the box
- restart your IDE
- right click on the copilot window
- select 'open devtools'
- a window should pop up with multiple tabs, select the console tab to view any errors / logs that are present

Do you have any other settings set in PyCharm that might be affecting this?

If you are willing, we can also hop on a call and debug this together live!

Thanks,
Caleb

**Comment by cf-home** [comment]
Hi Caleb,  restarting PyCharm temporarily resolved the issue.  I was able to reproduce the issue by moving the Pieces Copilot window to my laptop screen, then back to my larger external monitor screen (I stayed connected to the external monitor).  I don’t see any errors in my logs.

**Comment by caleb-at-pieces** [comment]
Hello @cf-home 

Good news and bad news, I have managed to reproduce this issue on my end, however it seems this is actually an issue with the Jetbrains IDE itself, and not the Pieces plugin.

From here I suggest that, while switching screens, or screen resolutions, make sure that the Copilot chat window is not opened in PyCharm. This includes changing the resolution of the screen Jetbrains is on, or moving Jetbrains to a new screen that has a different resolution that its current screen.

If the issue does happen again, restarting Jetbrains seems to resolve it.

Restarting Jetbrains can be done in a timely manner through:
- 'command + shift + a'
- search & select 'restart ide'

I will continue looking for a workaround for this on my end. Here is the youtrack issue I created for this: https://youtrack.jetbrains.com/issue/IJPL-156461

**Comment by cf-home** [comment]
﻿﻿Thank you Caleb, I appreciate how quickly you looked into this!On Jun 11, 2024, at 12:17 PM, Caleb Anderson ***@***.***> wrote:﻿
Hello @cf-home
Good news and bad news, I have managed to reproduce this issue on my end, however it seems this is actually an issue with the Jetbrains IDE itself, and not the Pieces plugin.
From here I suggest that, while switching screens, or screen resolutions, make sure that the Copilot chat window is not opened in PyCharm. This includes changing the resolution of the screen Jetbrains is on, or moving Jetbrains to a new screen that has a different resolution that its current screen.
If the issue does happen again, restarting Jetbrains seems to resolve it.
Restarting Jetbrains can be done in a timely manner through:

'command + shift + a'
search & select 'restart ide'

I will continue looking for a workaround for this on my end. Here is the youtrack issue I created for this: https://youtrack.jetbrains.com/issue/IJPL-156461
